tests/hazmat/primitives/test_aead.py

```diff
@@ -31,7 +35,7 @@ def test_chacha20poly1305_unsupported_on_older_openssl(backend):
 
 
 @pytest.mark.supported(
-    only_if=lambda backend: backend.chacha20poly1305_supported(),
+    only_if=lambda backend: backend.aead_cipher_supported(ChaCha20Poly1305),
```

Comment:
If we're switching to an `aead_cipher_supported` API, can we split that out into a separate PR (lol)